File: ip-messaging/rest/services/retrieve-service/retrieve-service.py
# Download the Python helper library from twilio.com/docs/python/install
from twilio.rest.ip_messaging import TwilioIpMessagingClient

# Your Account Sid and Auth Token from twilio.com/user/account
account = "ACCOUNT_SID"
token = "AUTH_TOKEN"
client = TwilioIpMessagingClient(account, token) 

# Retrieve the service
service = client.services.get(sid="SERVICE_SID")
print(service)

File: sync/rest/services/retrieve-service/retrieve-service.rb
require "http"
require 'twilio-ruby'

# Initialize the client
account_sid = 'ACCOUNT_SID'
auth_token = 'AUTH_TOKEN'
client = Twilio::REST::Client.new(account_sid, auth_token)

# Get the service
service = client.preview.sync.services('SERVICE_SID')
puts response